nobo.py

```diff
@@ -542,3 +550,26 @@ def get_current_zone_mode(self, zone_id, now=datetime.datetime.today()):
 
         self.logger.debug('Current mode for zone {} at {} is {}'.format(self.zones[zone_id]['name'], current_time, current_mode))
         return current_mode
+        
+    # Function to get temperature from a component
```

Comment:
Hi, do these functions work? I added them to the dev_comp_zone_temp branch, but I did not get a positive response from someone with a tempsensor